### DISCUSSION TITLE
win 11 任务栏混乱；Windows 11 wrong taskbar

### DISCUSSION THREAD

**Discussion by Airven131** [zhongyang219/TrafficMonitor#1144] [Q&A]
在更新win11 22598.1后任务栏窗口会变得混乱，无法显示在正确的位置；
After I update Windows 11 22598.1, the taskbar window will be wrong place.
如图所示；just like this:
![image](https://user-images.githubusercontent.com/93823547/163349333-552363a2-1608-4ff2-a8ea-36e820f9c648.png)

系统版本；system version:
![image](https://user-images.githubusercontent.com/93823547/163350079-ae774148-2a1b-4486-b729-afcb3cb4a065.png)

**Comment by MrZMZ** [comment]
我也是今天更新系统后出现了混乱，和你的状况一模一样。我是win11 22408-1503昨晚刚更新

**Reply by Airven131** [reply]
解决方案在 #1106
